tools/topology/topology2/cavs-nocodec.conf

```diff
@@ -83,11 +83,11 @@ Define {
 	SSP1_ENABLED			"true"
 	PASSTHROUGH			"false"
 	SSP0_PCM_NAME			"Port0"
-	SSP0_CAPTURE_PCM		"ssp-capture"
+	SSP0_CAPTURE_PCM		"Port0 2nd Capture"
```

Comment:
Why is this "2nd Capture" , I can't immediately figure out... UPDATE: ok, checked the #7093 .